sunpy/net/jsoc/jsoc.py

```diff
@@ -579,8 +582,8 @@ def get_request(self, requests, path=None, overwrite=False, progress=True,
         dl_set = True
         if not downloader:
             dl_set = False
-            downloader = Downloader(progress=progress, overwrite=overwrite)
-
+            downloader = Downloader(progress=progress, overwrite=overwrite, max_conn=4)
```

Comment:
max_conn should be treated the same as `max_splits`.

Reply:
max_conn goes to Downloader constructor. I don't think it can be passed as a kwarg. Can it be?

Reply:
it would have to be explicit (and not in `**kwargs`) I guess.